change/@azure-msal-browser-39c2410e-0746-4f81-bc8e-f8d9bbfce4cf.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Added unit tests for PlatformDOMRequest ExtraParameters (v5) #7825",
```

Changelogs should describe only what has changed in the published package
```suggestion
  "comment": "Logger statements updated #7825",
```